src/Core/Services/ConfigService.cs

```diff
@@ -10,32 +10,34 @@ namespace Bit.Core.Services
 {
     public class ConfigService : IConfigService
     {
-        private ConfigResponse _configs;
         private const int UPDATE_INTERVAL_MINS = 60;
+        private ConfigResponse _configs;
         private readonly IApiService _apiService;
         private readonly IStateService _stateService;
+        private readonly ILogger _logger;
 
-        public ConfigService(IApiService apiService, IStateService stateService)
+        public ConfigService(IApiService apiService, IStateService stateService, ILogger logger)
         {
             _apiService = apiService;
             _stateService = stateService;
+            _logger = logger;
         }
 
         public async Task<ConfigResponse> GetAllAsync()
         {
             try
             {
                 _configs = _stateService.GetConfigs();
-                if (_configs == null || _configs.ExpiresOn == null || _configs.ExpiresOn <= DateTime.UtcNow)
+                if (_configs?.ExpiresOn is null || _configs.ExpiresOn <= DateTime.UtcNow)
                 {
-                    _configs = await _apiService.GetAllConfigsAsync();
+                    _configs = await _apiService.GetConfigsAsync();
                     _configs.ExpiresOn = DateTime.UtcNow.AddMinutes(UPDATE_INTERVAL_MINS);
                     _stateService.SetConfigs(_configs);
                 }
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                //ignore and return state value or null
+                _logger.Exception(ex);
             }
```

Comment:
Just to make sure, do you want to also log the exceptions produced by no connection? I think they are not necessary and will pollute AppCenter logs but lmk so we can filter them out if not needed.

Reply:
Oh of course, i'll change it to not log those.